source/fem-dem/vans_assemblers.cc

```diff
@@ -1482,10 +1482,13 @@ GLSVansAssemblerMagnus<dim>::calculate_particle_fluid_interactions(
         {
           auto particle_properties = particle.get_properties();
 
+          // Add factor to rotational velocity to avoid divisions by zero
+          double omega_z =
+            particle_properties[DEM::PropertiesIndex::omega_z] + 1e-15;
+
           // Spin parameter
           double spin_parameter =
-            particle_properties[DEM::PropertiesIndex::dp] *
-            abs(particle_properties[DEM::PropertiesIndex::omega_z]) /
+            particle_properties[DEM::PropertiesIndex::dp] * abs(omega_z) /
```

Comment:
instead of having repeated abs everywhere, instead I would put the abs in a different variable (omega_norm) or something like this and use it.